packages/core/src/geometry/Geometry.ts

```diff
@@ -39,11 +40,20 @@ const map = {
  */
 export class Geometry
 {
+    public buffers: Array<Buffer>;
+    public indexBuffer: Buffer;
+    public attributes: {[key: string]: Attribute};
+    public id: number;
+    public instanced: boolean;
+    public instanceCount: number;
+    glVertexArrayObjects: {[key: number]: {[key: string]: WebGLVertexArrayObject}};
```

Comment:
Access modifiers

Reply:
we cannot make them explicit, becouse `glVertexArrayObjects` used in Renderer.
We can't mark it as private and as protected, becouse it hide it, but we can't mark it as public, becouse it is not for users.

Reply:
@eXponenta Which means they're essentially `internal`. Too bad TS doesn't have that. 

That brings the next question - [according to the docs](https://www.typescriptlang.org/docs/handbook/classes.html#public-by-default), the default access modifier is `public`, so they'll still appear in the generated documentation, right? Wouldn't it be clearer to mark them public and add add tsdoc `@private` with a few words of explanation why?